crates/viewer/re_view_spatial/src/visualizers/cameras.rs

```diff
@@ -77,10 +77,11 @@ impl CamerasVisualizer {
         let Some(pinhole_tree_root_info) = transforms.pinhole_tree_root_info(pinhole_frame_id)
         else {
             // This implies that the transform context didn't see the pinhole transform.
-            // Should be impossible!
-            return Err(
-                "Transform context didn't register the pinhole transform, but `CamerasVisualizer` is trying to display it!".to_owned(),
-            );
+            // This can happen with various frame id mismatches.
+            return Err(format!(
+                "Expected well-defined pinhole transforming into the transform frame {:?}, but look-up in active transform tree failed.",
```

Comment:
This error still feels pretty unspecific, but maybe we don't have more context to give here?